app/controllers/nonsipp/declaration/PsaDeclarationController.scala

```diff
@@ -67,12 +68,14 @@ class PsaDeclarationController @Inject()(
       schemeDateService.schemeDate(srn) match {
         case None => Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
         case Some(dates) =>
+          val now = schemeDateService.now()
           def emailFuture: Future[EmailStatus] =
```

Comment:
Do we really need this "extra" emailFuture here ? It seems it's just calling sendEmail only.

Reply:
I deliberately left it as it is in ER.

Reply:
We can refactor it later when everything works if you think it is necessary.

Reply:
so shall we remove it if it's not required then ?

Reply:
I think they are unnecessary :) okay, we can remove them later.